Makefile.docker

```diff
@@ -33,6 +33,8 @@ update-build-image-manifest: ## Update build image manifest and commits the chan
 	for i in .github/workflows/commit.yaml .circleci/config.yml ; do \
 	  sed "s|\(weaveworks/eksctl-build\):.*$$|\1:$(build_image_tag)|" "$${i}" > "$${i}.sedtmp" && mv "$${i}.sedtmp" "$${i}" ; \
 	done
+	git config --local user.email 'action@github.com'
+	git config --local user.name "Github Action"
```

Comment:
We don't actually push commit from GHA at the moment, but it's something we should probably do in the future. Right now this is expected to be called by a person that made a change to either `Dockerfile`, `go.mod` or `install-build-deps.sh`. All those files have appropriate comments in them.

Reply:
This is being ran by GHA, I saw it in the logs. That's why I knew that the `git commit` command on line 38 was failing. 

Look at .github/workflows/image-update.yaml:10 it is calling this part.